### DISCUSSION TITLE
Algo está a falhar

### DISCUSSION THREAD

**Discussion by guiworldtv2** [LITUATUI/M3UPT#25] [Q&A]
Estes links da rtp estão a funcionar para si?
rtp1 e rtp2 por exemplo, não estão a abir direto do brasil :(
algum problema com os comandos?

**Comment by LITUATUI** [comment]
Aqui em Portugal esses canais funcionam bem.

Durante a transmissão de alguns programas (filmes e desporto principalmente) o sinal deve ficar indisponível noutros países por causa dos direitos de transmissão.

**Reply by guiworldtv2** [reply]
Acredito que meus IP's estejam bloqueados na RTP PLAY, acessei através de uma VPN  e consegui acessar o site, minha rede está impossibilitada de acessar os sites da RTP :(

**Reply by LITUATUI** [reply]
Há cerca de duas semanas aconteceu o mesmo comigo, só consigo aceder aos canais da RTP se usar VPN. O problema já foi relatado no [chat do projecto](https://t.me/m3uptchat).
